elide-integration-tests/src/test/java/com/yahoo/elide/tests/FilterIT.java

```diff
@@ -1362,6 +1362,21 @@ void testFailFilterAuthorBookByChapter() throws IOException {
         assertNotNull(result.get("errors"));
     }
 
+    @Test
+    void testFailFilterBookByAuthorAddress() throws IOException {
+        /* Test default */
+        JsonNode result = getAsNode("book?filter[book.authors.homeAddress]=main&include=authors");
+        JsonNode data = result.get("data");
+        assertEquals(0, data.size(), result.toString());
+
+        /* Test RSQL */
+        result = getAsNode(
+                String.format("book?filter[book]=authors.homeAddress=='main'", hemingwayId),
+                HttpStatus.SC_BAD_REQUEST);
```

Comment:
Why does this give a bad request but the above tests returns an empty data?  I suspect the RSQL filter is formatted incorrectly.

Reply:
Bad merge.  Will fix that.